core/src/main/java/org/dst/core/table/Field.java

```diff
@@ -0,0 +1,103 @@
+package org.dst.core.table;
+
+
+public class Field {
+
+  private String name;
+
+  private ValueType type = ValueType.NONE;
+
+  private boolean primary = false;
+
+  private boolean index = false;
+
+  public Field() {
```

Comment:
Why we need this ctor?

Reply:
If not explicitly declared ,we can't use this structure function